src/esp/assets/ResourceManager.h

```diff
@@ -35,6 +35,11 @@
 #include "esp/gfx/configure.h"
 #include "esp/scene/SceneNode.h"
 
+// #include "managers/AssetAttributesManager.h"
```

Comment:
If you do not need these headers, would you consider removing them?

Reply:
These are for the next PR.  The code for the asset and object attribute managers is done, these are just stubs for them.